src/SprykerEco/Zed/AmazonQuicksight/Communication/Console/SyncCreateQuicksightUsersConsole.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AmazonQuicksight\Communication\Console;
+
+use Spryker\Zed\Kernel\Communication\Console\Console;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \SprykerEco\Zed\AmazonQuicksight\Persistence\AmazonQuicksightRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Business\AmazonQuicksightFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Communication\AmazonQuicksightCommunicationFactory getFactory()
+ */
+class SyncCreateQuicksightUsersConsole extends Console
+{
+    /**
+     * @var string
+     */
+    protected const COMMAND_NAME = 'quicksight-user:sync:create';
+
+    /**
+     * @var string
+     */
+    protected const COMMAND_DESCRIPTION = 'Creates quicksight users for registered quicksight users that can be matched with existing backoffice users from persistence.';
```

Comment:
It is not clear from the description that users are created in DB and they are compared with user on Quicksight side